### PR TITLE
Wb redis tests

### PR DESCRIPTION
**Pull request by kminevskiy**

### CODE REVIEW

File: lib/workerholic/queue.rb [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
   class Queue
-    attr_reader :storage, :name
+    attr_reader :storage, :name, :logger
 
-    def initialize(name = 'workerholic:queue:main')
+    def initialize(name = 'workerholic:queue:main', enable_logger = false)
       @storage = Storage::RedisWrapper.new
       @name = name
-      @logger = LogManager.new
+      @logger = enable_logger ? Logger.new : nil
```
Comment (timmy-lee) [inline lib/workerholic/queue.rb]: LogManager?

Comment (antoineLeclercq) [reply lib/workerholic/queue.rb]: agreed, also, not sure this is how we should handle the fact that we don't want to log out anything when running tests. This approach suggests we will need to make this modification on any class using an instance of `LogManager`. I think we should probably find a way to do this globally by defining a test environment.

Comment (kminevskiy) [reply lib/workerholic/queue.rb]: Yeah, I forgot to change it back. Global config option sounds good.

File: lib/workerholic/queue.rb [hunk 2]
```diff
@@ -11,7 +11,7 @@ def initialize(name = 'workerholic:queue:main')
 
     def enqueue(serialized_job)
       storage.push(name, serialized_job)
-      @logger.log('info', "Your job was placed in the #{name} queue on #{Time.now}.")
+      logger.log('info', "Your job was placed in the #{name} queue on #{Time.now}.")
```
Comment (timmy-lee) [inline lib/workerholic/queue.rb:14]: Need `@` or an `attr_reader` above.

**Review by timmy-lee — APPROVED** [review]

**Pull request merged**